internal/testing/check_api_change.sh

```diff
@@ -53,12 +52,19 @@ trap cleanup EXIT
 
 git clone -b "$UPSTREAM_BRANCH" . "$MASTER_CLONE_DIR" &> /dev/null
 
+# Copy the current branch into the temparory directory to avoid changing go.mod's
+cp -R . "$BRANCH_DIR"
```

Comment:
I'm not following why this is now needed over the old script.

Reply:
Turns out I might be doing something wrong. Removed, PTAL.